src/test/a11y/mocks.ts

```diff
@@ -63,3 +63,11 @@ mock('ccj/guards/ccjGuard', {
 mock('ccj/guards/individualDateOfBirthGuard', {
   'IndividualDateOfBirthGuard': justForwardRequestHandler
 })
+
+mock('response/guards/guardFactory', {
+  GuardFactory: {
```

Comment:
In the future this generic guard factory should help us keeping route specific guards close to the routes and extract guards to separate files only when these need to be shared across many routes